packages/web-service/src/pages/site/common/__tests__/site-map-upload.spec.js

```diff
@@ -73,16 +72,18 @@ describe('the map of the site showing the mitigations after development page pag
     const request = {
       cache: () => ({
         getData: () => ({
-          fileUpload: { filename: 'site-after-development.shp', path: '/tmp/path' },
```

Comment:
I do not believe that *.shape files are accepted --- look at 
packages/web-service/src/pages/common/file-upload/file-upload.js line 8